zulipterminal/helper.py

```diff
@@ -493,3 +493,95 @@ def notify(title: str, html_text: str) -> None:
     if command:
         res = subprocess.run(shlex.split(command), stdout=subprocess.DEVNULL,
                              stderr=subprocess.STDOUT)
+
+
+def parse_html_table(table_element: Any) -> Tuple[List[str], List[List[str]]]:
+    """
+    Parses an HTML table to extract cell items and column alignments.
+    """
+    headers = table_element.thead.tr.find_all('th')
+    rows = table_element.tbody.find_all('tr')
+    alignments = []
+
+    # The table cells are stored row-wise:
+    # [[row0, row0, row0],
+    # [row1, row1, row1],
+    # [row2, row2, row2]].
+    # Add +1 to count the header row as well.
```

Comment:
This looks like it's part of the comment above? I assume it is not?

Perhaps the cell description should go into the docstring?

Reply:
No, they are different. I'll move the upper part into the docstring.